_functions.sh

```diff
@@ -207,22 +207,28 @@ initialize_product_settings() {
 
   # ${PRODUCT_BRANCH} is computed from ${PRODUCT_VERSION} and is equal to the version up to the latest dot
   # and with x added. ex : 3.5.0-M4-SNAPSHOT => 3.5.x, 1.1.6-SNAPSHOT => 1.1.x
-  env_var PRODUCT_BRANCH `expr "${PRODUCT_VERSION}" : '\([0-9]*\.[0-9]*\).*'`".x"
-  env_var PRODUCT_MAJOR_BRANCH `expr "${PRODUCT_VERSION}" : '\([0-9]*\).*'`".x"
-  configurable_env_var "INSTANCE_ID" ""
+  if [[ -v PRODUCT_VERSION ]]; then
```

Comment:
The -v option is not available on macos. Is it not possible to use a classical test with  ``-z`` or ``-n`` ?

Why not using a test on PRODUCT_NAME == docker ?